ansible.cfg

```diff
@@ -4,14 +4,13 @@ stdout_callback=community.general.diy
 
 [callback_diy]
 ; Note that the action can be both k8s_info and community.kubernetes.k8s_info
-runner_retry_msg='
+runner_retry_msg="{% if ansible_callback_diy.result.output.attempts == 1 %}
```

Comment:
If we reach the retry limit, I'd also like to know what the state was. Can we also print more details if `attempts == retries?`

Reply:
At the end of the retries I expect to end either with OK or FAIL an in both cases I think  we are printing the state. Do you think there is a case when it should pass silently after the number of retries without OK or FAIL?

Reply:
Does it pretty print on failure? Or when it fails, does it print the state as JSON?

Reply:
I think we still have the same failure message as in this example https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_centos/490/consoleFull
Probably we can filter the failure message issue to be printed but I think failures need full message as in the example.

Reply:
For me, the YAML is much easier to read. I can see exactly which machine failed to provision in that example, and then I can look for it in the logs. So I think your change is a good improvement because we don't need to print over and over again when nothing has changed. But I would also like the easily readable summary at the end of the retries if something has gone wrong as well.